shared/utilities/ticket-status/index.js

```diff
@@ -1,6 +1,37 @@
 import moment from 'moment';
 import { isBefore, isAfter, formatDate } from '../date';
 
+const defaultOptions = [
+  {
+    title: '',
+    ticketsAvailable: false,
+    waitingListOpen: false,
+    displayLevel: 'Featured',
```

Comment:
Would this accidentally put half-written event information into the featured list?